pkg/api/server/register_images.go

```diff
@@ -624,6 +642,21 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//    description: |
 	//      output configuration TBD
 	//      (As of version 1.xx)
+	//  - name: inputStream
+	//    in: body
+	//    description: "A tar archive compressed with one of the following algorithms: identity (no compression), gzip, bzip2, xz."
```

Comment:
This parameter seems to provide the build context. I don't think there is another parameter to provide it. So without it you can't build an image.